GTBitsOfGood/access-h2o
# Backend for Client Information

## PR Title/Tagline

Backend linking for info submission page and note handling.

### How to Test

1. Pull to a local directory and execute npm run dev
2. use the prereview link provided in the below comments
NOTE: to test you should create dummy data in the database.
related databases:
infosubmissions
clients
notes

### Important Changes

-Add backend api for red and write data of info submission page and notes

### Related Github Issues

- #[92]
